app/components/primer/alpha/navigation_list/item.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= render(Primer::BaseComponent.new(tag: :li, **@system_arguments)) do %>
```

Comment:
since we always use `tag: :li` we can make it part of the `system_arguments` with `@system_arguments[:tag] = : li` and use `deny_tag_argument`